Cargo.toml

```diff
@@ -9,6 +9,8 @@ version = "0.1.10"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
```

Comment:
This needs the version changed on the top of the file as well. This shouldn't wipe out the approval though.